flash/core/utilities/apply_func.py

```diff
@@ -32,10 +33,4 @@ def get_callable_dict(fn: Union[nn.Module, Callable, Mapping, Sequence]) -> Unio
 
 
 def _is_overridden(method_name: str, instance: object, parent: Type[object]) -> bool:
```

Comment:
Hi, @uakarsh - As per @carmocca's suggestion [here](https://github.com/Lightning-AI/lightning-flash/pull/1457/files#r973777946), we can just use `lightning_utilities.core.overrides.is_overridden` wherever `_is_overridden` is used in Flash.

Doing a `grep -r "_is_overridden"` in flash, looks like it's only used in [flash/core/data/base_viz.py](https://github.com/Lightning-AI/lightning-flash/blob/master/flash/core/data/base_viz.py#L127). So all you will have to do is, replace `_is_overridden` there with `is_overridden` and add the import (`from lightning_utilities.core.overrides import`).

Once done, we can delete `_is_overridden` function in this file (apply_func.py).

In case it's not clear, please let me know and I can push these changes for you.

Reply:
I will surely do it now, but just wanted to ask, if you can see one test was not being passed (and it required), it was due to an error that said `OSError, sndfile is not installed`, I guess it is in flash.audio package and is depended upon lobrosa.

Reply:
You can ignore it for now (for this PR, I mean), I will take a look! Thanks for letting me know, @uakarsh

Reply:
Done!!